shared/test/test_properties.rb

```diff
@@ -29,6 +29,12 @@ def test_get_set_delete
     set_key_value(key, value_boolean)
     assert_equal value_boolean, get_key_value(key)
 
+    assert JSON.parse(list(@properties, @channel_id)).length == 1, "Owner can list all properties."
+
+    other_properties = Rack::Test::Session.new(Rack::MockSession.new(PropertiesApi, "studio.code.org"))
+    list(other_properties, @channel_id)
+    assert other_properties.last_response.unauthorized?, "Non-owner cannot list all properties."
```

Comment:
I'd prefer to see this ownership concept broken out into its own test case, e.g. `test_only_owner_can_list_all_properties`. These two assertions make a nice clean test by themselves, and it should be easy to extract since you already extracted the setup.

```ruby
  def test_only_owner_can_get_all_properties
    create_channel
    set_key_value('_testKey', 'one'.to_json)

    assert JSON.parse(list(@properties, @channel_id)).length == 1, "Owner can list all properties."

    other_properties = Rack::Test::Session.new(Rack::MockSession.new(PropertiesApi, "studio.code.org"))
    list(other_properties, @channel_id)
    assert other_properties.last_response.unauthorized?, "Non-owner cannot list all properties."

    delete_channel
  end
```